cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -0,0 +1,165 @@
+package workerqueue
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/pkg/errors"
```

Comment:
let's use `errors` over `github.com/pkg/errors`, as `github.com/pkg/errors` is not being actively developed, and most if not all features of that package have been imported into the standard `errors` package over years

Reply:
@neel-astro good catch and thanks. Will change to use the correct package.